### PR TITLE
feat(server/debug): introduce 'npwd-debug-dump' command

### PR DESCRIPTION
**Pull request by TasoOneAsia**
**Pull Request Description**

Introduces a debug command that will write out useful debug information regarding the environment NPWD is installed in, this should help ease support.

This also takes the chance to tweak the File transport logger to always write out to sv_npwd.log, no matter the debug level. Furthermore, Console transport is now tied directly to the config debug level, instead of inheriting from base logger instance.

**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [ ] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

File: resources/server/commands/registerCommands.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  const activePlayerCount = GetNumPlayerIndices();
+  mainLogger.info(`Connected Player Count > ${activePlayerCount}`);
+
+  try {
+    const playerTableResults = await DbInterface.fetch<ColumnInfo[]>(
+      `SELECT COLUMN_NAME, DATA_TYPE, CHARACTER_MAXIMUM_LENGTH, IS_NULLABLE
+       FROM INFORMATION_SCHEMA.COLUMNS
+       WHERE TABLE_NAME = '${config.database.playerTable}'`,
+    );
```
Comment (c-wide) [inline resources/server/commands/registerCommands.ts:34]: I could be wrong here but wont INFORMATION_SCHEMA contain data from other databases on the same server, unless you specify the database?

Comment (TasoOneAsia) [reply resources/server/commands/registerCommands.ts:34]: You're probably right, I forgot about that specifier when writing this. It has yet to be tested either.

**Comment by TasoOneAsia** [comment]
@itschip push changes you had made for this branch, por favor.

**Comment by itschip** [comment]
> push changes you had made for this branch, por favor.

yeah, I'll push it later today

**Comment by itschip** [comment]
@TasoOneAsia Let's try and merge this within the week.

**Pull request merged**